src/main/java/lotto/model/LottoNumbers.java

```diff
@@ -0,0 +1,24 @@
+package lotto.model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lotto.LottoMatchResult;
+import lotto.LottoNumber;
+
+public class LottoNumbers {
+
+    private final List<LottoNumber> lottoNumbers;
+
+    public LottoNumbers(List<LottoNumber> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public List<LottoMatchResult> guess(WinningNumber winningNumber) {
+        return lottoNumbers.stream().map(lottoNumber -> lottoNumber.match(winningNumber)).collect(Collectors.toList());
```

Comment:
객체지향 생활 체조 원칙의 `규칙 4: 한 줄에 점을 하나만 찍는다.`를 적용하면 가독성이 향상될 것 같습니다.
```suggestion
        return lottoNumbers.stream()
            .map(lottoNumber -> lottoNumber.match(winningNumber))
            .collect(Collectors.toList());
```